arm/Microsoft.AppConfiguration/configurationStores/deploy.bicep

```diff
@@ -157,6 +162,18 @@ resource configurationStore 'Microsoft.AppConfiguration/configurationStores@2021
   }
 }
 
+module appConfig_KeyValues 'keyValues/deploy.bicep' = [for (keyValues, index) in keyValuesList: {
```

Comment:
```suggestion
module configurationStore_keyValues 'keyValues/deploy.bicep' = [for (keyValues, index) in keyValuesList: {
```

Reply:
Changed 'appConfig_KeyValues' to 'configurationStore_keyValues'